tests/integration/security/fuzz/README.md

```diff
@@ -0,0 +1,41 @@
+# Fuzz test for authorization policy
+
+The directory includes the fuzz test for authorization policy to test potential policy bypass issues caused by path
+normalization issues in Istio and Envoy.
+
+This test is currently optional and not executed in the normal integration test. It is recommended to run the test
+before each new Istio release and make sure all fuzz tests pass.
+
+## Overview
+
+![](overview.jpg)
```

Comment:
What does this mean?

Reply:
This is just to show the image in markdown.